packages/components/psammead-navigation/src/index.jsx

```diff
@@ -0,0 +1,214 @@
+import React, { Fragment } from 'react';
+import styled, { css } from 'styled-components';
+import { shape, string, node, bool } from 'prop-types';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { C_WHITE, C_POSTBOX, C_GHOST } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MAX,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import { getPica, GEL_FF_REITH_SANS } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+
+/* White with 30% transparency over #B80000 */
+const BORDER_COLOR = '#eab3b3';
+
+const StyledNav = styled.nav`
+  padding: 0 ${GEL_SPACING};
+  background-color: ${C_POSTBOX};
+  position: relative;
+
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    padding: 0;
+  }
+`;
+
+const SkipLink = styled.a`
+  position: absolute;
+  clip-path: inset(100%);
+  clip: rect(1px, 1px, 1px, 1px);
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  padding: 0.75rem 0.5rem;
+  background-color: #ffffff;
+  border: 0.1875rem solid #000;
+  color: #333;
+  text-decoration: none;
+  font-family: ${GEL_FF_REITH_SANS};
+
+  &:focus {
+    clip-path: none;
+    clip: auto;
+    top: -3.75rem;
```

Comment:
Lots of random numbers here, can you make them variables and describe what they are? Are they dependancy on font size or lineheight?